qdrant-landing/content/documentation/private-cloud/configuration.md

```diff
@@ -103,7 +103,7 @@ operator:
     # Controller related settings
     controller:
       # The period a forced recync is done by the controller (if watches are missed / nothing happened)
-      forceResyncPeriod: 10h
+      forceResyncPeriod: 2m
```

Comment:
please do not change this value - it is going to invalidate the cache every 2mins. We reverted to 10h for good reasons back then.

Reply:
But that's the value we have been setting in private cloud.

Reply:
https://github.com/qdrant/qdrant-private-cloud/pull/25